executor/runner/runner.go

```diff
@@ -23,6 +24,10 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+var (
+	errTaskKilledBeforeCreation = errors.New("Task was killed before Task was created")
```

Comment:
Maybe we could formalize the steps or phases that the titus-executor does, to make it more obvious what "create" means in this context.